src/main.tsx

```diff
@@ -1,8 +1,16 @@
 import "./css/style.css";
+import "../node_modules/identity-style-guide/dist/assets/scss/_styles.scss";
+
 import { render } from "preact";
 import { Link } from "./router";
 import { Routes } from "./routes";
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as USWDS from "../node_modules/uswds/dist/js/uswds";
```

Comment:
I'd not expect to see USWDS referenced directly here, since we use (and depend on) `identity-style-guide`.

Might help with your other issues, but could we do one or the other of...

```suggestion
import "identity-style-guide/auto";
```

Or:

```suggestion
import { banner /*, ... */ } from "identity-style-guide";
[ banner /*, ... */ ].forEach((component) => component.on());
```

Ref:

- https://github.com/18F/identity-idp/blob/14364e5/app/javascript/app/components/index.js
- https://github.com/18F/identity-style-guide#javascript-package